cmd/commands/cli/command_identities.go

```diff
@@ -304,6 +311,91 @@ func (c *cliApp) setPayoutAddress(args []string) error {
 	return nil
 }
 
+const usageSetBeneficiary = "beneficiary-set <providerIdentity> <beneficiary>"
+
+func (c *cliApp) setBeneficiary(actionArgs []string) error {
+	if len(actionArgs) < 2 || len(actionArgs) > 3 {
+		clio.Info("Usage: " + usageSetBeneficiary)
+		return errors.New("malformed args")
+	}
+
+	address := actionArgs[0]
+	benef := actionArgs[1]
+	hermesID, err := c.config.GetHermesID()
+	if err != nil {
+		return err
+	}
+
+	err = c.tequilapi.SettleWithBeneficiary(address, benef, hermesID)
+	if err != nil {
+		return err
+	}
+
+	timeout := time.After(30 * time.Second)
```

Comment:
is beneficiary stored on the bc? If so shouldn't we also exit when it is in progress? waiting 30 secs seems a bit pointless no?

Reply:
Tranasction is made on polygon. Considering minimal transactor congestion it should complete in under 15 seconds.